cypress/support/hardhat.ts

```diff
@@ -0,0 +1,81 @@
+import { Eip1193Bridge } from '@ethersproject/experimental/lib/eip1193-bridge'
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { Wallet } from '@ethersproject/wallet'
+import { HardhatUtils, Network } from 'cypress-hardhat/lib/browser'
+
+export class HardhatProvider extends Eip1193Bridge {
```

Comment:
should this move to cypress-hardhat?

Reply:
probably. this used to live here because we did not fork from a separate package.
I'll take this as a TODO/follow-up for cypress-hardhat.